benchmarks/model-based/README.md

```diff
@@ -76,33 +76,145 @@ cd examples
 python analyze_experiment_results.py
 ```
 
-## Example benchmark
-
-<center>
-    <img style="border-radius: 0.3125em;
-    box-shadow: 0 2px 4px 0 rgba(34,36,38,.12),0 2px 10px 0 rgba(34,36,38,.08);"
-    src="./benchmarks/SafetyCarGoal1-v0-modelbased.png">
-    <br>
-    <div style="color:orange; border-bottom: 1px solid #d9d9d9;
-    display: inline-block;
-    color: #999;
-    padding: 2px;">SafetyCarGoal1-v0-modelbased</div>
-</center>
-
-<center>
-    <img style="border-radius: 0.3125em;
-    box-shadow: 0 2px 4px 0 rgba(34,36,38,.12),0 2px 10px 0 rgba(34,36,38,.08);"
-    src="./benchmarks/SafetyPointGoal1-v0-modelbased.png">
-    <br>
-    <div style="color:orange; border-bottom: 1px solid #d9d9d9;
-    display: inline-block;
-    color: #999;
-    padding: 2px;">SafetyPointGoal1-v0-modelbased</div>
-</center>
+## OmniSafe Benchmark
+
+Serving as a reliable SafeRL baseline, OmniSafe offers performance insights for SafeRL algorithms within the Safety-Gymnasium environment. The results of OmniSafe are presented in <a href="#performance_model_based">Table 1</a> and <a href="#curve_model_based">Figure 1</a>.
```

Comment:
```suggestion
Serving as a reliable SafeRL `baseline`, OmniSafe offers performance insights for SafeRL algorithms within the Safety-Gymnasium environment. The results of OmniSafe are presented in <a href="#performance_model_based">Table 1</a> and <a href="#curve_model_based">Figure 1</a>.
```
How do we comprehend baseline or benchmark's exact meaning? This makes me to associate Stable-Baselines3, I guess it wants to say: one algorithm which is classical is a baseline.

Reply:
As we use `baseline` in our paper, I just follow it. Some one suggests to me that do not use `benchmarks` so I use `baseline` instead.

Reply:
Currently, I have reached an agreement with @muchvo to use the term `experiment results` instead of `baseline`.